Services/JobService.cs

```diff
@@ -202,63 +210,26 @@ private string GetDietaryConsiderationsText(string familyId)
     private string GetPreviousRecipesText(string familyId)
     {
         var previousRecipes = _previousRecipeService.GetPreviousRecipes(familyId).Data!;
-        var enjoyedBreakfast = new List<string>();
-        var enjoyedLunch = new List<string>();
-        var enjoyedDinner = new List<string>();
-        var indifferentBreakfast = new List<string>();
-        var indifferentLunch = new List<string>();
-        var indifferentDinner = new List<string>();
-        var notEnjoyedBreakfast = new List<string>();
-        var notEnjoyedLunch = new List<string>();
-        var notEnjoyedDinner = new List<string>();
+        var enjoyed = new List<string>();
+        var notEnjoyed = new List<string>();
```

Comment:
How are we determining what meal is what on the overview page with these changes?

Reply:
Ohh are they just not sorted on the frontend now?

Reply:
the meatType is still there for the recipes but we just don't use them when building the prompt. all we are concerned about for each recipe is if they enjoyed them or not. for the overview page we still can access the mealtype for sorting purposes

Reply:
Ahh gotcha, nice